source/__tests__/integration/import-profile.test.js

```diff
@@ -159,17 +157,20 @@ test('imports valid v0.2.0 profile from json file', async () => {
 })
 
 test('imports propertyURI value with # char cleanly', async () => {
-  expect.assertions(3)
-
+  expect.assertions(4)
+  await page.$eval('form[name="profileForm"]', e => e.reset())
   await page.goto('http://localhost:8000/#/profile/create/true')
   const profilePath = path.join(__dirname, "..", "__fixtures__", 'profile_needs_encoding_v0.1.0.json')
   await expect(page).toUploadFile('input[type="file"]', profilePath)
   const rtemplateSpanSel = 'span[popover-title="Resource ID: sinopia:resourceTemplate:bf2:Place"]'
-  await page.waitForSelector(rtemplateSpanSel)
+  await page.waitForSelector(rtemplateSpanSel, {visible: true})
   await expect(page).toClick(rtemplateSpanSel)
-
-  // NOTE: this propertyURI has #
   const ptemplateSpanSel = 'span[popover-title="Property URI: http://www.w3.org/2000/01/rdf-schema#label"]'
-  await expect(page).toMatchElement(ptemplateSpanSel)
-  // shameless green:  could not figure out a way to select the value in the actual input element
+  await page.waitForSelector(ptemplateSpanSel, {visible: true})
+  await expect(page).toClick(ptemplateSpanSel)
+
+  const propURISel = 'input[name="propertyURI"]'
+  await page.waitFor(1000) // shameless green: it needs more time here; can't tell why
```

Comment:
Curious if you tried collapsing lines 173 and 174, like:

```javascript
await page.waitForSelector(propURISel, { timeout: 1000 })
```

Reply:
oh yeah, good call, would be interested to see whether that worked.

does the `page.waitFor(1000)` call reset a global wait config value, or is that just like "pause for 1000 ms on this page right now"?

Reply:
so `page.waitFor(1000)` waits for one second - a "slow this test down here no matter what" (yuck).  `await page.waitForSelector(propURISel, { timeout: 1000 })` waits for a max of one second:  https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagewaitforselectorselector-options -- basically, waitForSelector on its own should work, as it defaults to 30 seconds.  But it doesn't work. 

In this spot, I tried many things, including waiting for 500 ms instead of 1000, etc.  This is the way it worked, hence the comment.

Reply:
so it goes.  works for me.  (esp since it's not setting something globally, and just pausing in the one test)

Reply:
Thanks for explaining! :Shipit: